### ISSUE TITLE
Read the bins.csv from HDFS-restructure script.

### ISSUE THREAD

**Issue by blootsvoets** [RADAR-CNS/RADAR-RestApi#35]
The `bins.csv` file generated by the HDFS restructure script currently has entries as follows:

```csv
topic,device,timestamp,count
application_uptime,6771326b-36ff-4aec-9593-9525379152f8,20170727_15,81
```

The REST API could read these and serve them so we have an idea how many records have been added.

**Comment by yatharthranjan** [comment]
For this to be available, the `hdfs_restructure.sh` has to be run at regular intervals to produce upto date information. Is that right ? So how can we run this script via the REST-API or maybe can add it to systemd timers to run it periodically ?

**Comment by blootsvoets** [comment]
In radar-docker, a systemd timer for hdfs_restructure is installed with the install-systemd-wrappers script.

**Comment by yatharthranjan** [comment]
Okay. Thanks!

**Comment by yatharthranjan** [comment]
closed by #62
